data/World/Africa/ZAF/trignet.json

```diff
@@ -0,0 +1,33 @@
+[
+    {
+        "name": "TrigNet",
+        "description": "South African National Geospatial Information (NGI) Network",
+        "urls": [
+            "http://196.15.132.2:2101"
+        ],
+        "reference": {
+            "url": "ftp://ftp.trignet.co.za/TrigNet Presentations/TRIGNET Presentation 2024.pptx",
+            "comments": "http://www.trignet.co.za/"
```

Comment:
Should this be `https`?

Reply:
Have you tried it? :rofl:
Ask them, not me!